scripts/local-dev.sh

```diff
@@ -306,12 +343,21 @@ start_local_services() {
     echo -e "${BLUE}🎨 Starting Streamlit UI...${NC}"
     (cd src/ui && \
       MCP_SERVER_URL="http://localhost:$MCP_PORT" \
-      streamlit run ui.py --server.port $UI_PORT --server.address 0.0.0.0 --server.headless true) &
+      PYTHON_LOG_LEVEL="$PYTHON_LOG_LEVEL" \
+      streamlit run ui.py --server.port $UI_PORT --server.address 0.0.0.0 --server.headless true > /tmp/summarizer-ui.log 2>&1) &
```

Comment:
Using PYTHON_LOG_LEVEL to be passed from cli, and output is redirected to a file in `/tmp`